Makefile

```diff
@@ -40,7 +40,8 @@ help: ## Display this help.
 
 .PHONY: manifests
 manifests: controller-gen ## Generate WebhookConfiguration, ClusterRole and CustomResourceDefinition objects.
-	$(CONTROLLER_GEN) rbac:roleName=manager-role crd webhook paths="./..." output:crd:artifacts:config=build/yaml/crd/
+	$(CONTROLLER_GEN) rbac:roleName=manager-role crd webhook paths="./pkg/apis/v1alpha1/" output:crd:artifacts:config=build/yaml/crd/
+	$(CONTROLLER_GEN) rbac:roleName=manager-role crd webhook paths="./pkg/apis/v1alpha2/" output:crd:artifacts:config=build/yaml/crd/
```

Comment:
why we have tow different version: apis/v1alpha1 and apis/v1alpha2?
Also, I noted there is one more new folder pkg/apis/nsx.vmware.com? what is purpose of this folder?

Reply:
Talk as discussed. IPPool has 2 version.

Reply:
do we need to update yaml as well?

Reply:
Should we put the CRD of NCP defined earlier in pkg/apis/v1alpha1/? Otherwise, when `make manifests`, there is only v1alpha2 ippool in build/yaml/crd/nsx.vmware.com_ippools.yaml.
I manually added the v1alpha1 in the yaml, so it would override it, and only v1alpha2 was left behind. @timdengyun @heypnus @dantingl

Reply:
I think if we need two versions of  ippool, we might need to put it in both.  if that, make manifests also need to put two versions' yaml in the different in build/yaml/crd, otherwise, the nsx.vmware.com_ippools.yaml will be override by the v1alpha2.

Reply:
As discussed, we can only keep v1alpha2 for temp.